x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -204,12 +210,26 @@ function NodeDetailLink({ id, name, nodeID }: { id: string; name?: string; nodeI
 }
 
 // eslint-disable-next-line react/display-name
-const NodeDetailTimestamp = memo(({ eventDate }: { eventDate: Date | undefined }) => {
-  const formattedDate = useFormattedDate(eventDate);
+const NodeDetailTimestamp = memo(
+  ({ eventDate, id }: { eventDate: Date | undefined; id: string }) => {
+    const formattedDate = useFormattedDate(eventDate);
 
-  return formattedDate ? (
-    <CopyablePanelField textToCopy={formattedDate} content={formattedDate} />
-  ) : (
-    <span>{'—'}</span>
-  );
-});
+    return formattedDate ? (
+      <SecurityCellActions
+        data={{
+          field: '@timestamp',
+          value: formattedDate,
+        }}
+        triggerId={SecurityCellActionsTrigger.DEFAULT}
+        mode={CellActionsMode.HOVER_DOWN}
+        sourcererScopeId={getSourcererScopeId(id)}
+        metadata={{ scopeId: id }}
+        disabledActionTypes={[SecurityCellActionType.FILTER, SecurityCellActionType.TOGGLE_COLUMN]}
```

Comment:
why disable filter?

Reply:
@kqualters-elastic there was concern of showing filter in/filter out in the rule creation workflow and whether it will lead to strange behaviors in timeline (i.e. filter added to global kql bar vs. timeline)

Upon some testing:
- Filter in/out work correctly in alert table ✅
- In timeline, filters are applied to timeline ✅
- In rule creation workflow/alert preview, it's not an issue because other elements in timeline has filter actions and we don't have a preview timeline... 

Having the filters give user consistent experience so I've added them back. cc @paulewing